agent/cache-types/service_checks.go

```diff
@@ -75,6 +75,9 @@ func (c *ServiceHTTPChecks) Fetch(opts cache.FetchOptions, req cache.Request) (c
 			return hash, reply, nil
 		},
 	)
+	if err != nil {
+		return result, fmt.Errorf("LocalBlockingQuery: %v", err)
```

Comment:
I think `LocalBlockingQuery` is an internal implementation detail that probably won't be relevant to whoever ends up receiving this error message. 

I took a quick look at `Agent.LocalBlockingQuery()` and it seems the only return value will come from the function we pass in as the 4th parameter. All of those errors already have context added to them, so I think in this return we don't have any additional context to add. Returning the error unmodified should be fine:

```go
if err != nil {
    return result, err
}
```